chromium/background.js

```diff
@@ -320,12 +320,14 @@ function onBeforeRequest(details) {
   // If there is a username / password, put them aside during the ruleset
   // analysis process
   var using_credentials_in_url = false;
+  let tmp_user;
```

Comment:
var for consistence

Reply:
`let` for ES6.

Reply:
not even consistent in the same function ...

Reply:
Will be when we will run ESLint with new settings.